### PR TITLE
Prod fixes

### LINKED ISSUE #311: Safeguard production (deploy & rollback)
We had an almost 12 hour quest creator outage caused by a simple code bug that spiraled out of control due to deployment automation misconfiguration and failure in procedure.

See the [postmortem doc](https://docs.google.com/document/d/1SUpPxQua-tMB7TwMBcK0tXTlgjBQxg0gUzEMOLZnuRg/edit?usp=sharing) for followup action items.

### PR DESCRIPTION
**Pull request by smartin015**
Fixes #311

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -143,6 +142,7 @@
     "webpack-dev-server": "^2.4.2",
     "yargs": "^7.0.2"
   },
+  "devDependencies": {},
   "engines": {
     "node": "^6.9.0"
```
Comment (toddmedema) [inline package.json]: this should be node 8 now, correct?

File: package.json [hunk 2]
```diff
@@ -82,9 +83,7 @@
     "typo-js": "^1.0.3",
     "warning": "^3.0.0",
     "webpack": "^2.2.1",
-    "winston": "^2.3.1"
-  },
-  "devDependencies": {
```
Comment (toddmedema) [inline package.json]: We can switch these back to being dev dependencies and not use heroku postbuild / resume our normal deploy process, correct? (unless we really don't care about the distinction between dev and prod dependencies)

Comment (smartin015) [inline package.json]: Done

File: package.json (hunk 1 above)
Comment (smartin015) [inline package.json]: Done

**Comment by smartin015** [comment]
Made a bunch of fixes (including docs) and set up master so that it only pushes to dev.  PTAL

File: webpack.static.config.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+// Use this for building static error/maintenance pages.
```
Comment (toddmedema) [inline webpack.static.config.js]: Why is this separate from prod deployment config? Don't we want to always deploy these and decide whether to show them in the code?

Comment (smartin015) [inline webpack.static.config.js]: Fix'd. Note that we should eventually setup travis to autopush these pages to S3.

**Pull request merged**